src/olympia/lib/settings_base.py

```diff
@@ -85,10 +85,11 @@ def path(*folders):
 CLEANCSS_BIN = env(
     'CLEANCSS_BIN', default='node_modules/less/bin/lessc')
 
-# Path to uglifyjs (our JS minifier).
+# Path to our JS minifier.
 # Set as None to use YUI instead (at your risk).
```

Comment:
Is this comment still valid? I didn't see anything in `_minify` that suggests this, nor anywhere else that `JS_MINIFIER_BIN` is used.

Reply:
Looks like this was indeed removed in https://github.com/mozilla/addons-server/pull/8694